content/styles/03-components/header.css

```diff
@@ -8,10 +8,12 @@ header {
     justify-content: space-between;
     align-items: center;
     position: fixed;
+    padding: 1rem 9rem;
     top: 0;
     left: 0;
     width: 100%;
-    z-index: 9;
+    /* overide leaflet and notification */
```

Comment:
great to have an explanation here 👌

Reply:
Do you think this kind of exception should be stay in neomad not styleguide? I really hesitate to set it on this repo.

Reply:
Not quite sure yet. Do as you feel for now and we'll adjust later if necessary.